src/Tests/InstrEngineTests/NaglerInstrumentationMethod/InstrumentationEngineApi.cpp

```diff
@@ -52,10 +52,11 @@ namespace InstrumentationEngineApi
 #else
             // Aid in finding the line of code associated with the error message:
             // stderr: The specified module could not be found.
-            wchar_t err[256];
+            WCHAR err[256];
             memset(err, 0, 256);
             FormatMessageW(FORMAT_MESSAGE_FROM_SYSTEM, NULL, GetLastError(), MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), err, 255, NULL);
-            AssertLogFailure(err);
+            LPCWSTR errMsg = err;
```

Comment:
Well, I suppose that's a way to get around the constness problem.

You could also use a //lgtm comment to make it explicit instead:

[https://help.semmle.com/lgtm-enterprise/user/help/alert-suppression.html#format-of-suppression-comments](https://help.semmle.com/lgtm-enterprise/user/help/alert-suppression.html#format-of-suppression-comments)

This isn't really doing anything of interest that actually does anything for security because AssertLogFailure is const on the callee side.

Reply:
Ah well next time I'll use LGTM comments for these types of issues.